### PR TITLE
Podspec: include headers in nested directories

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

File: ios/CocoaPods/Filament.podspec [hunk 1]
```diff
@@ -50,12 +50,12 @@ Pod::Spec.new do |spec|
   end
 
   spec.subspec "gltfio_core" do |ss|
-    ss.source_files = "include/gltfio/*.h"
+    ss.source_files = "include/gltfio/**/*.h"
```
Comment (romainguy) [inline ios/CocoaPods/Filament.podspec:53]: gltfio doesn't have nested directories for its public headers

Comment (alexey-pelykh) [reply ios/CocoaPods/Filament.podspec:53]: https://github.com/google/filament/blob/main/libs/utils/include/utils/Mutex.h#L23 that looks like include of a nested directory to me

Comment (romainguy) [reply ios/CocoaPods/Filament.podspec:53]: That's in `utils`, not `gltfio`.

Comment (alexey-pelykh) [reply ios/CocoaPods/Filament.podspec:53]: 🤦‍♂️ sorry, you're right. Let me adjust that

Comment (alexey-pelykh) [reply ios/CocoaPods/Filament.podspec]: done

**Comment by romainguy** [comment]
Thanks! I'll let @bejado finish the review because he's our iOS expert. I don't see any issues but I'm not familiar with podspecs so… :)

**Review by bejado — APPROVED** [review]
Thank you!

**Pull request merged**